projects/packages/search/src/instant-search/components/gridicon/index.jsx

```diff
@@ -228,6 +228,7 @@ class Gridicon extends Component {
 
 		return (
 			<svg
+				aria-label={ this.props?.description }
```

Comment:
I think we can safely assume a React component will have a `this.props` so probably don't need the optional chaining here (I notice we access this.props a few lines below for focusable).
```suggestion
				aria-label={ this.props.description }
```